locales/de/calendar.json

```diff
@@ -0,0 +1,48 @@
+{
+  "date": "Datum",
+  "time": "Uhrzeit",
+  "event": "Formel 1 Grand Prix Veranstaltungen",
```

Comment:
Would need context from devs to chooser closer fitting translation

Reply:
This is used as a hidden title just above the table of the calendar events. Let me know if I can help with extra context.

Reply:
ah I see. This is fine then.